source/includes/crud/Delete.java

```diff
@@ -24,14 +27,26 @@ public static void main(String[] args) {
             MongoDatabase database = mongoClient.getDatabase("sample_mflix");
             MongoCollection<Document> collection = database.getCollection("movies");
 
-            Bson query = lt("imdb.rating", 1.9);
+            Bson deleteOneQuery = eq("title", "The Garbage Pail Kids Movie");
+
+            try {
+                // Deletes the first document that has a "title" value of "The Garbage Pail Kids Movie"
+                DeleteResult result = collection.deleteOne(deleteOneQuery);
+                System.out.println("DeleteOne document count: " + result.getDeletedCount());
```

Comment:
S: I'd change to "Delete one" or "deleteOne()"
```suggestion
                System.out.println("deleteOne() document count: " + result.getDeletedCount());
```